### PR TITLE
Replace div with button and adjust styling

### PR DESCRIPTION
**Pull request by kimallen**
## Description

This replaces the div with a button element.  It adapts the styles to match the original styles and removes the `onKeyPress` handling that the button handles automatically.

## Reviewer Notes

I made the button background color transparent-- should i make it match the color of the background instead?

I'm not sure how div height was determined, but the default button height was off, so I made it close as possible with the div height (div was 51px, but since we do dimensions divisible by 4, I made it 52px)

## Setup

Go to a Customer that has multiple shipments (`nts@ntsr.unsubmitted `) or create multiple shipments.  See the list of shipments on the Home page.  The visual should not be different than before.  I added cursor pointer as well.

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4929) for this change

## Screenshots

image

### CODE REVIEW

File: src/components/Customer/Home/ShipmentList/ShipmentList.module.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
   padding-right: 13px;
   padding-top: 12px;
   padding-bottom: 12px;
+  width: 100%;
+  height: 52px;
+  background-color: transparent;
+  cursor: pointer;
   @include u-margin-bottom(1);
 
   strong {
```
Comment (Shkeating) [inline src/components/Customer/Home/ShipmentList/ShipmentList.module.scss:20]: image
I'm noticing the next in the `strong` tag is not left aligned like they were before. To remediate this, add `text-align: left;` to your scss for the `strong`text in this component.

**Review by Shkeating — CHANGES_REQUESTED** [review]

**Review by suzubara — APPROVED** [review]
other than @Shkeating 's comment, looks good to me!

File: src/components/Customer/Home/ShipmentList/ShipmentList.module.scss (hunk 1 above)
Comment (kimallen) [reply src/components/Customer/Home/ShipmentList/ShipmentList.module.scss:20]: Nice- fixed!

**Review by Shkeating — APPROVED** [review]
looks good to me!

looks good!

**Pull request merged**